include/opengemini/impl/cli/Functor.hpp

```diff
@@ -19,6 +19,8 @@
 
 #include "opengemini/impl/ClientImpl.hpp"
 
+#include "opengemini/Query.hpp"
```

Comment:
Query should be on the top of `ClientImpl`

Reply:
The order of includes file we follows:
1. Related header
2. C/C++ standard library headers
3. System library headers
4. Third-party library headers
5. Current project's headers